src/test/groovy/org/gradle/profiler/studio/launcher/StudioConfigurationProviderTest.groovy

```diff
@@ -41,16 +41,19 @@ some other text
             studioInstallDir.toPath().resolve("lib/lib2.jar"),
             studioInstallDir.toPath().resolve("lib/lib1/lib1.jar")
         ]
+
+        where:
+        classpathKeyword << ["CLASS_PATH", "CLASSPATH"]
```

Comment:
Let's document somewhere which keyword belongs to the old vs. new versions.

Reply:
Good idea, added a comment.